cmd/cluster-agent/api/v1/install.go

```diff
@@ -33,10 +37,64 @@ func init() {
 
 // Install registers v1 API endpoints
 func Install(r *mux.Router, sc clusteragent.ServerContext) {
+	// The /metadata endpoints are deprecated. They will be removed as of 1.0.
+	// Agents < 6.5.0 are using /metadata.
 	r.HandleFunc("/metadata/{nodeName}/{ns}/{podName}", getPodMetadata).Methods("GET")
 	r.HandleFunc("/metadata/{nodeName}", getNodeMetadata).Methods("GET")
 	r.HandleFunc("/metadata", getAllMetadata).Methods("GET")
+	r.HandleFunc("/tags/pod/{nodeName}/{ns}/{podName}", getPodMetadata).Methods("GET")
+	r.HandleFunc("/tags/pod/{nodeName}", getNodeMetadata).Methods("GET")
+	r.HandleFunc("/tags", getAllMetadata).Methods("GET")
```

Comment:
I would think that this endpoint returns both `pod` tags and `node` tags but currently it only returns `pod` tags. Should this endpoint be

```
r.HandleFunc("/tags/pod", getAllMetadata).Methods("GET")
```

I think a `/tags` endpoint still makes sense though but maybe in another PR and could have similar output to the agent's `tagger-list`.